src/components/Search.js

```diff
@@ -28,14 +30,30 @@ const Search = props => (
                 />
               </td>
               <td>
-                <button className="btn btn-primary fn-submit-name">Search IDs</button>
+                <SearchButton />
+              </td>
+              <td>
```

Comment:
Do you see a pattern here? Would it be useful to extract out some components to make it more readable?